core/controllers/profile.py

```diff
@@ -89,8 +89,15 @@ def get(self, username):
             edited_exp_summary_dicts = (
                 summary_services.get_displayable_exp_summary_dicts_matching_ids(
                     user_contributions.edited_exploration_ids))
+            ### function to determine whether current user is same as user on page ###
```

Comment:
For comments in Python a `#` at the beginning of the line is sufficient. For this, the code is self explanatory that a comment isn't necessary.